Gemfile.lock

```diff
@@ -1,14 +1,94 @@
 PATH
   remote: .
   specs:
-    bump (0.5.3)
+    bump (0.5.12)
```

Comment:
@grosser good catch, the test specs were executing against the project Gemfile.lock